projects/core/src/occ/config-loader/occ-config-loader.module.ts

```diff
@@ -12,12 +13,20 @@ import { OccConfigLoaderService } from './occ-config-loader.service';
  */
 export function initConfig(
   configLoader: OccConfigLoaderService,
-  config: SiteContextConfig
-): ConfigInitializer {
+  config: SiteContextConfig,
+  /**
+   * @deprecated since 3.2, TODO: remove it in 4.0
+   */
```

Comment:
Not public API. No need to deprecate.

Instead, please deprecate the this whole module with notion: `use SiteContextConfigLoaderModule instead`